SS/server_old.R

```diff
@@ -0,0 +1,320 @@
+rm(list = ls())
```

Comment:
if this is going to be moved into an SS folder, just move the old file without a rename - everything is tracked via commits anyways